### ISSUE TITLE
 incompatible peer dependency to "@angular/core" (requires "^8.0.0" (extended), would install "10.0.4")

### ISSUE THREAD

**Issue by ofirrifo** [crisbeto/angular-svg-round-progressbar#202]
I'm get this error while I'm trying to upgrade to angular 10
```
  Package "angular-svg-round-progressbar" has an incompatible peer dependency to "@angular/core" (requires "^8.0.0" (extended), would install "10.0.4").
                  Package "angular-svg-round-progressbar" has an incompatible peer dependency to "@angular/common" (requires "^8.0.0" (extended), would install "10.0.4").

``` 
while I run the > ng update @angular/core @angular/cli

**Comment by ofirrifo** [comment]
I want to create a PR with package.json update fir latest angular version. 

I get push failed.

```
12:02	Push failed
			ERROR: Permission to crisbeto/angular-svg-round-progressbar.git denied to ofirrifo.
			Could not read from remote repository.
			
			Please make sure you have the correct access rights
			and the repository exists.
```

Please give me permission to commit and create a PR.
Thanks

**Comment by crisbeto** [comment]
I've published version 4.0.0 that updates the Angular dependencies to 10.
